main.py

```diff
@@ -0,0 +1,92 @@
+from fastapi import FastAPI, Request
+import uvicorn
+from cryptography.fernet import Fernet
+from os import popen
+
+app = FastAPI()
+from fastapi.responses import StreamingResponse
+
+
+def load_key():
+    """
+    Load the previously generated key
+    """
+    return open("secret.key", "rb").read()
+
+
+def encrypt_message(message):
+    """
+    Encrypts a message"
+    :param message: a regular message
+    :return: the encrypted message
+    """
+    key = load_key()
+    f = Fernet(key)
+    message = message.encode()
+    encrypted_message = f.encrypt(message)
+    return encrypted_message
+
+
+def decrypt_message(encrypted_message):
+    """
+    Decrypts an encrypted message
```

Comment:
Notice the grammar of your functions. Use grammar as to describe "What the function should do when called". The answer should be the title of the function. 

This means that you should document `decrypt_message` with `"Decrypt given message"` instead of `"Descrypts ..."`. Notice that I don't add the `s` at the end because I use a different tense
Fix everywhere.